esp/services/ws_fs/ws_fsBinding.cpp

```diff
@@ -370,6 +300,54 @@ IPropertyTree* CFileSpraySoapBindingEx::createPTreeForXslt(const char* method, c
     return pRoot.getClear();
 }
 
+//For every ECLWatchVisible dropzones, read: dropZoneName, directory, and, if available, computer.
+//For every Servers/Server in ECLWatchVisible dropzones, read: directory,
+// server name, and hostname(or IP). Create dropzone("@name", "@directory", "@computer",
+// "@netAddress", "@linux", "@sourceNode") tree into pSoftware.
+void CFileSpraySoapBindingEx::appendDropZones(double clientVersion, IConstEnvironment* env, const char* dfuwuidSourcePartIP, IPropertyTree* softwareTree)
+{
+    IArrayOf<IConstTpDropZone> list;
+    CTpWrapper dummy;
+    dummy.getTpDropZones(clientVersion, nullptr, true, list);
```

Comment:
This is what we do for now: 
                if (machine->getOS() == MachineOsLinux || machine->getOS() == MachineOsSolaris)
                {         
                    dir.replace('\\', '/');//replace all '\\' by '/'
                    pDropZone->setProp("@linux", "true");
                }
                else
                {       
                    dir.replace('/', '\\');
                    dir.replace('$', ':');
                }